server/tests/employer.form.test.js

```diff
@@ -273,6 +274,13 @@ describe('Server V1 Form Endpoints', () => {
     expect(res1.nonHcapHires).toEqual('1');
   });
 
+  it('checks response from the site participants endpoint', async () => {
+    const [site1] = await getSiteByID(1);
+    const res = await getHiredParticipantsBySite(site1.siteId);
+    // Two participants were added in the previous test
```

Comment:
I trust this works but am not a huge fan of using it. Tests should be able to be parallelized and/or run in an arbitrary order. May be another part of the upcoming test refactor.

Reply:
Yeah I would not trust the order at all

Repetitive code is fine, that's where helpers come into play to help each test set up the data/conditions it needs :)